### DISCUSSION TITLE
Search for other field than firstname and lastname in People search

### DISCUSSION THREAD

**Discussion by Jean-Pierre-Komans** [microsoft-search/pnp-modern-search#3668] [Q&A]
Hi,

I have created an employee directory using SharePoint Search. It works fine. I can search for the fields ie lastname, aboutme and first name, but if I search for a mobile phone number, I get no results. When I go in debug mode I see the field is available with the phonenumber and value.

How do I configure search to be able to find for example WorkPhone and MobilePhone. These values are available (i see them in debug mode).

Any help would be great :-).

Kind Regards,

JP

**Comment by kasperbolarsen** [comment]
You should be able to search for WorkPhone and/or MobilePhone but it is often tricky, as per https://www.m365thinking.com/post/searching-for-phone-numbers-in-the-employee-directory
Did this answer you question ?

**Reply by Jean-Pierre-Komans** [reply]
Hi,

Thanks for the reply. I have tried to change the query, but how does the query must look like. 

Now I use: {searchTerms} workemail:example.com to get all the people. 

How should I change the query?

**Comment by kasperbolarsen** [comment]
I guess you have a Search Box connected to the Results web part? In that case you could just enter the phone number in the search box. However, you have to type the phone number exactly as it shows up in the debug, so if the number in 23 34 56 78 you must type "23 34 56 78" , otherwise you will not get a hit
